### PR TITLE
Sync with MiMo42's version

### PR DESCRIPTION
**Pull request by barijaona**
Port of MiMo42/MMTabBarView#67
Further work to reduce differences with Mimo42/MMTabBarView

### CODE REVIEW

File: MMTabBarView/MMTabBarView/MMAttachedTabBarButton.m [hunk 1]
```diff
@@ -188,22 +188,28 @@ - (NSRect)draggingRect {
     return draggingRect;
 }
 
+inline static NSBitmapImageRep* imageForView(NSView* const inView, NSRect const inBounds) {
+	if (@available(macOS 10.14, *)) {
```
Comment (TAKeanice) [inline MMTabBarView/MMTabBarView/MMAttachedTabBarButton.m]: Just for readability, we should put the remainder of the non-10.14 code into an else-branch.

File: MMTabBarView Demo/MMTabBarView Demo/en.lproj/DemoWindow.xib [hunk 2]
```diff
@@ -531,7 +532,7 @@
                                     <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                                 </textFieldCell>
                             </textField>
-                            <popUpButton verticalHuggingPriority="750" id="9">
+                            <popUpButton verticalHuggingPriority="750" misplaced="YES" id="9">
```
Comment (TAKeanice) [inline MMTabBarView Demo/MMTabBarView Demo/en.lproj/DemoWindow.xib]: These misplaced attributes are like cancer in the xib-files. We should not include them anywhere if avoidable

File: MMTabBarView/MMTabBarView/MMAttachedTabBarButton.m (hunk 1 above)
Comment (barijaona) [reply MMTabBarView/MMTabBarView/MMAttachedTabBarButton.m]: I also noticed that, but I hesitated as the goal was to reduce differences with @MiMo42 's version.
Changed as you suggested.

File: MMTabBarView Demo/MMTabBarView Demo/en.lproj/DemoWindow.xib (hunk 2 above)
Comment (barijaona) [reply MMTabBarView Demo/MMTabBarView Demo/en.lproj/DemoWindow.xib]: Same remark. Changed now.

**Comment by TAKeanice** [comment]
I understand the changes done in order to come closer to MiMo42s version. Maybe, in a next step, we should check if there are any _functional_ incompatibilities, and if not, propose our version to MiMo in a PR.

**Pull request merged**